services-camel/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,12 @@
+
+appender.file.type = File
+appender.file.name = file
+appender.file.fileName = target/camel-test.log
```

Comment:
This path doesn't exist after deploying to the VM does it?  I think I recall getting an error about being unable to open a log file when starting up the service.  Maybe comment out the file appender for the time being?